src/place.c

```diff
@@ -153,9 +154,22 @@ static void parse_vertical_wall_string(const char *part, bool *leftwall, bool *r
 	parse_square_content(part[2], st);
 }
 
-static void init_place(struct Place *pl, const char *path)
+static void print_place_info(const struct Place *pl)
+{
+	log_printf("    path = %s", pl->path);
+	log_printf("    size %dx%d", pl->xsize, pl->zsize);
+	log_printf("    %d walls", pl->nwalls);
+	log_printf("    %d enemies that never die", pl->nneverdielocs);
+	log_printf("    enemies go to (%.2f, %.2f, %.2f)", pl->enemyloc.x, pl->enemyloc.y, pl->enemyloc.z);
+	for (int i = 0; i < 2; i++)
+		log_printf("    player %d goes to (%.2f, %.2f, %.2f)", i, pl->playerlocs[i].x, pl->playerlocs[i].y, pl->playerlocs[i].z);
+}
+
+static void read_place_from_file(struct Place *pl, const char *path)
 {
-	misc_basename_without_extension(path, pl->name, sizeof(pl->name));
+	log_printf("Reading place from '%s'...", path);
+	snprintf(pl->path, sizeof pl->path, "%s", path);
```

Comment:
Truncated path wouldn't be great, but also shouldn't happen because these are relative paths into `assets` folder:

```suggestion
	SDL_assert(strlen(path) < sizeof pl->path);
	strcpy(pl->path, path);
```